packages/protocol/contracts/rrp/authorizers/interfaces/IApi3TokenLock.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+interface IApi3TokenLock {
+    event SetMinimumLockingTime(uint256 minimumLockingTime);
+
+    event SetLockAmount(uint256 lockAmount);
+
+    event BlockRequester(
```

Comment:
Maybe events should be in past tense for consistency with other contracts: 

`event BlockedRequester(`

Same with Lock, Unlock, etc

Reply:
Done